ios/CodePush/RCTConvert+CodePushInstallMode.m

```diff
@@ -1,5 +1,10 @@
 #import "CodePush.h"
+
+#if __has_include("RCTConvert.h")
```

Comment:
Hi guys. I'm just solving probably the [issue](https://github.com/mauron85/react-native-background-geolocation/issues/44) with ordering, like you did here. Can you please explain why ordering in if ... else statement is important? Just don't get it, but seems to be causing duplicate imports. Many thanks.